packages/select-react/src/Select.tsx

```diff
@@ -362,9 +362,17 @@ export const Select = forwardRef<HTMLSelectElement, SelectProps>((props, forward
                     e.stopPropagation();
                     focusSelected(listElement, selectedValue);
                 }
+            } else if (e.key === "Enter" && dropdownIsShown) {
+                // This component already deviates from how a native select would react
+                // to Enter keypresses (a native <select> would submit the form, not open itself)
```

Comment:
Etter hva jeg har sett, så er ikke dette helt riktig. Da jeg testet en helt ren native select så submitter ikke den form hverken når den er åpen eller den har fokus. Forskjellen her er vel at man egentlig er i en <input>? 

Uansett riktig oppførsel, bare ikke helt 100% beskrivelse

Reply:
Tror det mest naturlige (for meg) ville vært at enter velger merket valg (og lukker menyen), men ikke gjør noe hvis det ikke er et valg merket. Men det som er lagt inn her er uansett mye bedre enn en uventet innsending av skjema!

Reply:
Kanskje vi bare burde lukke den dersom søkefeltet er tomt?

Reply:
En native select submitter formen dersom formen har en button med type "submit" om jeg ikke husker mye feil. Testet du en form som ikke hadde knapp? Evnt hvilken browser?